src/topp/FileConverter.cpp

```diff
@@ -446,7 +446,7 @@ class TOPPFileConverter :
                                                  CONVERSION_MZXML));
       MzXMLFile f;
       f.setLogType(log_type_);
-      ChromatogramTools().convertChromatogramsToSpectra<MSExperimentType>(exp);
+      //ChromatogramTools().convertChromatogramsToSpectra<MSExperimentType>(exp);
```

Comment:
basically, no :)
If we want to keep this functionality, we should make it XOR, i.e. either spectra or chromatograms, but not both. Would that work for your use cases?